### PR TITLE
Week3 nguyentrangiakiet

### PR DESCRIPTION
**Pull request by GiaKiet21522258**

### CODE REVIEW

File: .gitignore
Comment (hsangUIT) [inline .gitignore:1]: Please resolve conflict

File: Latex/conference_101719.pdf
Comment (hsangUIT) [inline Latex/conference_101719.pdf:1]: Please resolve conflict

**Review by hsangUIT — CHANGES_REQUESTED** [review]
Please resolve conflict

File: Latex/conference_101719.pdf
Comment (GiaKiet21522258) [reply Latex/conference_101719.pdf:1]: đã fix

**Pull request merged**